File: LevelEditor/code/models/Obj.ts
import * as _ from "lodash";
import { Point } from "./point";

export class Obj {
  name: string;
  spriteOrImage: string;
  snapToTile: boolean;
  snapOffset: Point;
  constructor(name: string, image: string, snapToTile: boolean, snapOffset: Point) {
    this.name = name;
    this.spriteOrImage = image;
    this.snapToTile = snapToTile;
    this.snapOffset = snapOffset;
  }


}

File: LevelEditor_CS/LevelEditor_CS/Models/Obj.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LevelEditor_CS.Models
{
    public class Obj
    {
        public string name;
        public string spriteOrImage;
        public bool snapToTile;
        public Point snapOffset;

        public Obj(string name, string image, bool snapToTile, Point snapOffset)
        {
            this.name = name;
            this.spriteOrImage = image;
            this.snapToTile = snapToTile;
            this.snapOffset = snapOffset;
        }


    }
}
